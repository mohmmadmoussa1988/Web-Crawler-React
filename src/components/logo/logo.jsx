import React from 'react';
import logo from '../../assets/Sixt-Logo.svg.png';
import {LogoImage} from './logo.styles';
import { Link } from "react-router-dom";

const Logo = () =>{
    return(
       <Link to="/"><LogoImage src={logo} /></Link>
    )
}

export default Logo;