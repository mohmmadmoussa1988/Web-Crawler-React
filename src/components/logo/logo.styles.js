import styled from 'styled-components'
export const LogoImage = styled.img`
width: 150px;
margin:0 auto;
`;
