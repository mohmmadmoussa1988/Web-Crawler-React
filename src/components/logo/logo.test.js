import React from 'react';
import {configure,shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Logo from './logo';
import { Link } from "react-router-dom";
import logo from '../../assets/Sixt-Logo.svg.png';
import toJson from 'enzyme-to-json';

configure({adapter:new Adapter()});
describe(`Logo`,()=>{

    it('should render image with link to home page',()=>{
        const wrapper = shallow(<Logo />);
        expect(toJson(wrapper)).toMatchSnapshot();
    })


})