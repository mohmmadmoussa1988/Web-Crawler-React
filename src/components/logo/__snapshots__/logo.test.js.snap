// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render image with link to home page 1`] = `
<Link
  to="/"
>
  <styled.img
    src="Sixt-Logo.svg.png"
  />
</Link>
`;
